e2e-tests/playwright/docs/accessibility/automated_scan_testing.md

```diff
@@ -0,0 +1,145 @@
+- **Color and Contrast** - WCAG AA/AAA contrast ratios
+- **Alternative Text** - Image alt text presence
+- **Table Structure** - Headers, captions, relationships
+- **Navigation** - Skip links, landmarks, focus order
+
+### **Cannot Be Fully Automated**
+
+- **Cognitive Accessibility** - Content readability, comprehension
+- **Contextual Appropriateness** - Alt text quality, meaningful link text
+- **Real User Experience** - Actual keyboard workflows, screen reader UX
+- **Content Quality** - Clear instructions, effective error messaging
+
+## Testing Patterns
+
+### **1. Component Testing**
+
+```typescript
+test('modal accessibility', async ({axe, page}) => {
+    await page.click('[data-testid="open-modal"]');
+
+    const results = await axe.builder(page).include('[role="dialog"]').analyze();
+
+    expect(results.violations).toHaveLength(0);
+});
+```
+
+### **2. State Testing**
+
+```typescript
+test('form error state accessibility', async ({axe, page}) => {
+    // Test normal state
+    let results = await axe.builder(page).include('form').analyze();
+    expect(results.violations).toHaveLength(0);
+
+    // Test error state
+    await page.click('[type="submit"]');
+    results = await axe.builder(page).include('form').analyze();
+    expect(results.violations).toHaveLength(0);
+});
+```
+
+## Rule Management
+
+### **When to Disable Rules**
+
+**Disable Rules Only When:**
+
+- **Known UI Framework Limitations** - Theme-related contrast issues
+- **Context-Inappropriate Rules** - Page-level rules in modal dialogs
+- **Temporary Workarounds** - With clear documentation and tracking
+
+### **Examples**
+
+```typescript
+// Good - Documented limitation
+test('dark theme modal', async ({axe, page}) => {
+    const results = await axe
+        .builder(page)
+        .include('[role="dialog"]')
+        .disableRules([
+            'color-contrast', // TODO: MM-nnn - Color contrast improvement
+        ])
+        .analyze();
+});
+
+// Good - Context on comment
+test('modal dialog', async ({axe, page}) => {
+    const results = await axe
+        .builder(page)
+        .include('[role="dialog"]')
+        .disableRules([
+            'page-has-heading-one', // Not applicable to modals
+            'landmark-one-main', // Not applicable to modals
+        ])
+        .analyze();
+});
+```
+
+**Don't Disable Rules For:**
+
+- **Convenience** - "This rule is annoying"
+- **Lack of Understanding** - "I don't know what this rule does"
+- **Time Pressure** - "We'll fix it later" (without tracking)
+
+## **Expected Coverage**
+
+**Overall: ~85% of accessibility requirements fully automated**
+
+| Area                 | Coverage |
+| -------------------- | -------- |
+| ARIA & Semantics     | 95%      |
+| Forms & Validation   | 90%      |
+| Interactive Elements | 85%      |
+| Color & Contrast     | 80%      |
+| Content Structure    | 95%      |
+
+**Approach:** Default comprehensive analysis with selective rule disabling only when necessary.
```

Comment:
Do we need this section? Unless we're going to be seriously trying to target these metrics, it feels like it's kind of unnecessary